janusgraph-core/src/main/java/org/janusgraph/core/JanusGraphFactory.java

```diff
@@ -75,6 +77,10 @@ public static JanusGraph open(String shortcutOrFile) {
         return open(getLocalConfiguration(shortcutOrFile));
     }
 
+    public static JanusGraph open(String shortcutOrFile, String backupName) {
```

Comment:
Need some javadoc especially for `backupName` parameter.

Reply:
This and the below method are public so they're already exposed. I agree with @robertdale that they should be documented. You can indicate that they're not intended to be used directly in the documentation.